README.md

```diff
@@ -1,2 +1,442 @@
+
+14.	Unpack the download archive file under /xampp/htdocs/drupal/modules.
+
+15.	In the web browser go back to the Drupal admin panel ‘localhost:5555/drupal/’.
+
+16.	Click the ‘Modules’ navigation button.
+
+17.	Scroll down on the modules page or find ‘H5P’ using the browser’s search function.
+
+18.	Select both ‘H5P’ and ‘H5P Editor’ and click the ‘Save Configuration’ button underneath.
+
+19.	Click the ‘Configure’ navigation button on the top navbar, then click on H5P under the system configuration group, scroll down, and enable both:
+
+a.	H5P development mode
+
+b.	Library development directory (for programmers only)
+
+20.	If everything is working correctly you should see the Interactive Content type when you click Add Content from the top menu.
+
+
+
+
+
+1. Got to the (C:\xampp\htdocs\drupal\sites\default\files\h5p\development),
+
+2. Clone the repo here
+   ```sh
+   git clone https://github.com/curiouslearning/FeedTheMonsterH5P.git
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
+4. Run 
+```sh
+npm run dev
+```
+in the terminal.
+
+
+25. Open Any Browser and go to http://localhost/drupal/.
+
+26. Click on Add content,then click on interactive content there select the content type having the same title provided in library.json.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+### Creating a new language version
+ 
+For adding new languages the older json shall be replaced by downloading the Language specific json file from
+
+ ```https://github.com/curiouslearning/ftm-asset-api/tree/main/fulljsons``` 
+
+and replace the older json from  
+
+```C:\xampp\htdocs\drupal\sites\default\files\h5p\development\FeedTheMonsterH5P\src\data\example-return.js.```
+
+then save and  compile it.
+
+Here's is code snippet from ```C:\xampp\htdocs\drupal\sites\default\files\h5p\development\FeedTheMonsterH5P\src\data\example-return.js```for the Reference
+
+
+``` const gameData = {
+  GeneralData: {
+    LanguageName: "English",
+    LanguageVersion: 44,
+    ImageBasedRendering: false,
+    GeneralAudio: {
+      GreatAudio: "http://server.com/audio/english/feedback/great.wav",
+      FantasticAudio: "http://server.com/audio/english/feedback/fantastic.wav",
+      A: "https://curiousreader.org/wp-content/uploads/2022/05/USENGLISH_sounds_letters_a.wav",
+    },
+    GeneralImages: {
+      GreatImage: "http://server.com/images/english/feedback/great.png",
+      FantasticImage: "http://server.com/images/english/feedback/fantastic.png",
+    },
+  },
+  FeedbackTexts: ["Fantastic!","Great!","Amazing!"],
+  
+
+  // Paste the language specific json here
+
+
+  };
+
+export default gameData;
+```
+
+### Compile and build the new version
+
+Step1 : For Compiling newer versions go to the library.json file and increase the "patchVersion" by 1 in both the 
```

Comment:
Consistency with numbering and styling here-- continue 1. ... , 2. ... , etc. format used above.